apis/Google.Cloud.Storage.V1/docs/index.md

```diff
@@ -57,6 +57,26 @@ Or write-only access to put specific object content into a bucket:
 
 {{sample:UrlSigner.SignedURLPut}}
 
+### Supported credential types for URL signig
+
+`Google.Apis.Auth.OAuth2.ServiceAccountCredential`, `Google.Apis.Auth.OAuth2.ComputeCredential`
+and `Google.Apis.Auth.OAuth2.ImpersonatedCredential` are all supported credentials from
+which you can build a `UrlSigner` by calling the appropiate `UrlSigner.FromCredential`
+method overload. `Google.Apis.Auth.OAuth2.GoogleCredential` is also supported as long as the
+underlying credentials is one of the supported specific types. The following example demonstrates
+how to explicitly use a Compute credential for URL signing.
+
+{{sample:UrlSigner.ComputeSignedURLGet}}
+
+When using a `UrlSigner` take care to choose the appropiate sync or async version of the signing
```

Comment:
I think "take care" is probably a little too strong. How about "it's worth being aware of the underlying synchronous/asynchronous nature" or something similar. (And I'd expand "sync" to "synchronous" everywhere, although "async" is probably okay.)